### PR TITLE
* added method to get field keys

### PR DESCRIPTION
**Pull request by wesokes**
@wesleykendall please review

### CODE REVIEW

File: kmatch/kmatch.py [hunk 1]
```diff
@@ -152,3 +152,28 @@ def match(self, value):
                 suppress_key_errors class variable is False
         """
         return self._match(self._compiled_pattern, value)
+
+    def get_field_keys(self, pattern=None):
+        """
+        Builds a set of all field keys used in the pattern including nested fields.
```
Comment (micahhausler) [inline kmatch/kmatch.py]: Can you add the description and type here?
```
:param pattern: <description>
:type pattern: <type>
```

**Comment by micahhausler** [comment]
Could you also add a note in `/docs/release_notes.rst` ?

**Comment by micahhausler** [comment]
@wesokes Also if you don't mind: can you drop Python 3.2 from `.travis.yml` and the `setup.py`'s classifiers and also add Python 3.5 to both those places?

**Comment by wesokes** [comment]
@micahhausler addressed your comments

**Comment by micahhausler** [comment]
LGTM!

**Pull request merged**